edit-post/components/sidebar/post-schedule/index.js

```diff
@@ -3,31 +3,48 @@
  */
 import { __ } from '@wordpress/i18n';
 import { PanelRow, Dropdown, Button } from '@wordpress/components';
+import { withInstanceId } from '@wordpress/compose';
 import { PostSchedule as PostScheduleForm, PostScheduleLabel, PostScheduleCheck } from '@wordpress/editor';
+import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
-export function PostSchedule() {
+export function PostSchedule( { instanceId } ) {
 	return (
 		<PostScheduleCheck>
 			<PanelRow className="edit-post-post-schedule">
-				<span>{ __( 'Publish' ) }</span>
+				<label
+					htmlFor={ `edit-post-post-schedule__toggle-${ instanceId }` }
+					id={ `edit-post-post-schedule__heading-${ instanceId }` }
+				>
+					{ __( 'Publish' ) }
+				</label>
 				<Dropdown
 					position="bottom left"
 					contentClassName="edit-post-post-schedule__dialog"
 					renderToggle={ ( { onToggle, isOpen } ) => (
-						<Button
-							type="button"
-							className="edit-post-post-schedule__toggle"
-							onClick={ onToggle }
-							aria-expanded={ isOpen }
-							isLink
-						>
-							<PostScheduleLabel />
-						</Button>
+						<Fragment>
+							<label
+								className="edit-post-post-schedule__label"
+								htmlFor={ `edit-post-post-schedule__toggle-${ instanceId }` }
+							>
+								<PostScheduleLabel /> { __( 'Click to change' ) }
+							</label>
+							<Button
+								id={ `edit-post-post-schedule__toggle-${ instanceId }` }
+								type="button"
```

Comment:
While preexisting, this line is redundant since `'button'` is already the default for `type`.

Reply:
Coolio, will remove.